app-emulation/retroarch/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
gentoo proxy maintainers project should be specified last, as you are not an official developer

Reply:
Moved author to end of file.